applications/plugins/SofaPython/python/SofaPython/Quaternion.py

```diff
@@ -35,6 +35,9 @@ def angle(q):
     """get angle in rad"""
     return 2.0* math.acos(re(q))
 
+def product(qa, qb):
```

Comment:
If it's a dot product, then call it `dot`? 

There are (at least) three potential products with quaternions.